wcfsetup/install/files/acp/update-com.woltlab.wcf_3.2_reactionUpdate.php

```diff
@@ -0,0 +1,58 @@
+<?php
+use wcf\data\like\Like;
+use wcf\data\option\OptionEditor;
+use wcf\system\WCF;
+
+// !!!!!!!!!
+// HEADS UP!    The columns for wcf1_like, wcf1_like_object and the wcf1_reaction_type table must already exists, before calling this script.
+// HEADS UP!    The foreign key for the wcf1_like table will be created within this script, after providing real values for the reactionTypeID
+// HEADS UP!    column. Also this script provides the basic reactionTypes.
+// !!!!!!!!!
+
+/**
+ * @author	Joshua Ruesweg
+ * @copyright	2001-2018 WoltLab GmbH
+ * @license	GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @package	WoltLabSuite\Core
+ */
+
+OptionEditor::import([
+	'like_show_summary' => 1,
+	'like_enable_dislike' => 0
+]);
+
+try {
+	WCF::getDB()->beginTransaction();
+	
+	// add reaction columns 
+	$statement = WCF::getDB()->prepareStatement(str_replace('wcf1_', 'wcf'.WCF_N.'_', 'INSERT INTO wcf1_reaction_type (title, type, showOrder, iconFile) VALUES (\'wcf.reactionType.title1\', 1, 1, \'like.svg\'), (\'wcf.reactionType.title2\', 1, 2, \'haha.svg\'), (\'wcf.reactionType.title3\', -1, 3, \'sad.svg\'), (\'wcf.reactionType.title4\', 0, 4, \'confused.svg\'), (\'wcf.reactionType.title5\', 1, 5, \'thanks.svg\')'));
```

Comment:
Consider using an HEREDOC for legibility.

Reply:
Splitting the query in `n` queries or only provide the values in HEREDOC and imploding them in the query?

Reply:
Single query is fine, use a HEREDOC and add a newline after each comma.